### ISSUE TITLE
Safezone bug - Damage inside safezone

### ISSUE THREAD

**Issue by TEKKONIC** [DraconisCluster/DX-Issues#52]
** tl; dr **
--Damage within the medina safe zone--

** Full info **
--player testimony--
I was in the Medina safe zone, connected to the station. DNB player, ScarryRabbit showed up with pretty rough attempt to stop his ship and suddenly his ship appeared in the middle of mine, without any damage. He tried to do fixship while I was moving forward and it helped. but I've lost control and bumped his ship soon after. And this time his drive was destroyed and also one of my cargo containers. So both ships were inside safe zone bubble but both ships took damage as if we were in normal space. After further inspection, I found that the first clash also damaged my ship, but deep inside. Probably in the same spot that Rabbit's ship was stuck in, somewhere in the middle of one of my reactors

**Screenshots**
--no screenshots or footage as of this time if they can find any i will provide it here--
